mysql-test/suite/innodb/t/innodb-page_compression_none.test

```diff
@@ -0,0 +1,30 @@
+--source include/have_innodb.inc
+--source include/have_sequence.inc
+
+--echo #
+--echo # MDEV-30825 innodb_compression_algorithm=0 (none) increments Innodb_num_pages_page_compression_error
+--echo #
+
+SET GLOBAL innodb_compression_algorithm=0, innodb_checksum_algorithm=strict_full_crc32, innodb_buffer_pool_size=8*1024*1024;
+SELECT VARIABLE_VALUE INTO @compress_errors FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME = 'Innodb_num_pages_page_compression_error';
+
+
+CREATE TABLE t (c INT) page_compressed=1 page_compression_level=4 ENGINE=InnoDB;
+INSERT INTO t VALUES (1);
+SET GLOBAL innodb_checksum_algorithm=crc32;
+CREATE TABLE t1 (c INT) page_compressed=1 page_compression_level=4 ENGINE=InnoDB;
+INSERT INTO t1 VALUES (1);
+
+# do stuff to purge it out of innodb buffer pool - TODO how to do better.
+FLUSH TABLES;
+CREATE TEMPORARY TABLE b AS SELECT seq, REPEAT('aabbccddeeffgghhiijjkkllmmnn', 100000) FROM seq_1_to_1000;
```

Comment:
So, this was the reason for the buffer pool size change?
I do not know if it would work reliably in 10.4, but starting with 10.5, there is a test `mysql-test/suite/innodb/t/page_cleaner.test` that should work.
You could also simply try
```sql
FLUSH TABLES t, t1 FOR EXPORT;
UNLOCK TABLES;
```